R/pkg/R/utils.R

```diff
@@ -851,3 +851,12 @@ rbindRaws <- function(inputData){
   out[!rawcolumns] <- lapply(out[!rawcolumns], unlist)
   out
 }
+
+# Get basename without extension from URL
+basenameSansExtFromUrl <- function(url) {
```

Comment:
can we use file_path_sans_ext [1] for removing the extension ? I worry we might publish it as `.tar.gz` someday and then removing just the last `.` will be insufficient

[1] https://stat.ethz.ch/R-manual/R-patched/library/tools/html/fileutils.html